testing/upload_validation.ts

```diff
@@ -885,7 +886,19 @@ const genericDynamicSyncTableSchema = zodCompleteObject<
   getSchema: formulaMetadataSchema,
 }).strict();
 
-const syncTableSchema = z.union([genericDynamicSyncTableSchema, genericSyncTableSchema]);
+const syncTableSchema = z.union([genericDynamicSyncTableSchema, genericSyncTableSchema])
+  .superRefine((data, context) => {
+    const syncTable = data as SyncTable;
+
+    if (syncTable.getter.varargParameters) {
```

Comment:
Just confirming that we don't need to check if this is an empty list here. Not sure if our definition wrappers spit out empty lists for fields that are omitted in the hand-authored definition.

Reply:
Done.